skrub/_expressions/_skrub_namespace.py

```diff
@@ -719,12 +858,24 @@ def eval(self, environment=None):
             expression are used. If a dict, it must map the name of each
             variable to a corresponding value.
 
+        keep_subsampling : bool, default=False
+            If True, and if subsampling has been configured (see
+            :meth:`Expr.skb.subsample_previews`), use a subsample of the data. By
+            default subsampling is not applied and all the data is used.
+
         Returns
         -------
         result
             The result of running the computation, ie of executing the
             pipeline's ``fit_transform`` on the provided data.
 
+        See Also
+        --------
+        Expr.skb.preview :
+            Access the preview of the result on the variables initial values,
+            with subsampling. Faster than ``eval`` but does not allow passing
+            new data and always applies subsampling.
```

Comment:
Can you make it more clear on how `preview` differs from `subsample_preview` (it's not clear in my head, I must admit)?